modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/UsersEndpoint.java

```diff
@@ -511,7 +545,7 @@ private Set<JpaRole> parseJsonRoles(final String roles) throws IllegalArgumentEx
     return rolesSet;
   }
 
-  private Map<String, Object> generateJsonUser(User user) {
+  public Map<String, Object> generateJsonUser(User user) {
```

Comment:
Is there a reason changing this to public? I didn't find a usage outside this class.

Reply:
The original idea was to use this function in the event and series endpoints as well, as they now send user data. That did not end up working out. Changed it per your suggestion.